src/components/ReviewConfirm/IncorporationDateTime.vue

```diff
@@ -297,6 +277,15 @@ export default class IncorporationDateTime extends Mixins(DateMixin) {
       .toLocaleTimeString('en-US', { hour: '2-digit', minute: '2-digit', hour12: true })
   }
 
+  private get isUnderTime (): boolean {
+    if (this.incorporationDateTime.effectiveDate) {
+      const startDate = new Date()
+      const effectiveDate = this.incorporationDateTime.effectiveDate
+      const diff = (effectiveDate.getTime() - startDate.getTime()) / 1000
```

Comment:
I believe since you are just using getTime() you can use `Date.now()` instead of `startDate.getTime()` so it doesn't have to instantiate a date object.

Reply:
Interesting, i'll try this out!

Reply:
Cool, i like it 👍 Thanks!